src/controllers/satisfactionSurvey/satisfactionSurvey.js

```
@@ -0,0 +1,604 @@
+        respondent.satisfactionSurveyResponses.push({
+          id: q.id,
+          question: q.question,
+          answer: null,
+        });
+      }
+      // Set start time
+      await respondent.set({ startedAt: new Date() });
+      await respondent.save();
+    }
+    const finalData = { ...survey.toJSON(), respondent };
+    delete finalData.createdAt;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)